qs-telemetry-dashboard/Helpers/CertificateConfigHelpers.cs

```diff
@@ -20,13 +20,15 @@ internal static string Hostname
 		{
 			get
 			{
-				if (string.IsNullOrEmpty(_hostname))
+				if (_hostname == null)
 				{
-					string hostnameBase64 = File.ReadAllText(FileLocationManager.HOST_CONFIG_PATH);
-					byte[] data = Convert.FromBase64String(hostnameBase64);
-					_hostname = Encoding.ASCII.GetString(data);
+
+					byte[] certArray = File.ReadAllBytes(@"C:\ProgramData\Qlik\Sense\Repository\Exported Certificates\.Local Certificates\client.pem");
+
+					_certificate = new X509Certificate2(certArray);
```

Comment:
this potentially breaks the other certificate property depending on when it's run.